include/lsst/afw/coord/Coord.h

```diff
@@ -131,7 +131,7 @@ class Coord {
     void rotate(Coord const &axis, lsst::afw::geom::Angle const theta);
     lsst::afw::geom::Angle offset(lsst::afw::geom::Angle const phi, lsst::afw::geom::Angle const arcLen);
     
-    Coord::Ptr convert(CoordSystem system) const;
+    Coord::Ptr convert(CoordSystem system, double epoch=2000) const;
```

Comment:
I don't see a way to comment directly after your remark about the 10% speedup below, so I'll do it here. I am not sure what you mean by "swapped sky axes"; there is no swapping involved in this speedup; it simply handles the overwhelmingly most common case that both WCS will have the same sky system. (It is fortunate it is so common since it was the ONLY case that was warped correctly!). I agree it may not be common to warp without interpolation, but the speedup makes interpolation a bit less crucial. Since you are willing to leave it in, I'm going to do so.